ambry-clustermap/src/main/java/com/github/ambry/clustermap/HelixBootstrapUpgradeUtil.java

```diff
@@ -863,22 +877,30 @@ private void addUpdateInstances(String dcName, Map<String, Set<String>> partitio
             // the appropriate instanceConfig that is constructed with the correct values from both).
             // For now, only bootstrapping cluster is allowed to directly change InstanceConfig
             dcAdmin.setInstanceConfig(clusterName, instanceName, instanceConfigFromStatic);
+          } else {
+            info(
```

Comment:
Minor: formating, info("[{}] Instance {} already present in Helix...",

Reply:
Actually it has been formatted by intellij, as I did option+cmd+L.